### PR TITLE
Robert/create market and orders

### PR DESCRIPTION
**Pull request by justinbarry**

### CODE REVIEW

**Comment by pgebheim** [comment]
@justinbarry -- is this a WIP?

ALSO do you happen to know what the original issue # is for this I can't find it.

**Comment by pgebheim** [comment]
@bthaile -- This PR is what we were looking at for new create market and orders scripts that don't need augur node.

**Comment by bthaile** [comment]
Yup, I made some changes on it to have better looking order books for dai.

**Comment by pgebheim** [comment]
@justinbarry I fixed merge conflicts. Wasn't it on your plate to get the `flash` cli working again somehow?

**Comment by justinbarry** [comment]
Ref #1421

File: packages/augur-test/src/flash/create-canned-markets-and-orders.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ async function placeOrder(person: ContractAPI,
                           price: DecimalBigNumber) {
   const tickSize = can.tickSize
     ? new DecimalBigNumber(can.tickSize)
-    : numTicksToTickSize(new DecimalBigNumber("10000"), new DecimalBigNumber("0"), new DecimalBigNumber("0x0de0b6b3a7640000"));
+    : numTicksToTickSize(new DecimalBigNumber("100"), new DecimalBigNumber("0"), new DecimalBigNumber("0x0de0b6b3a7640000"));
```
Comment (bthaile) [inline packages/augur-test/src/flash/create-canned-markets-and-orders.ts:106]: Will this bake in the 3 decimal places for price? Meaning traders and market makers will only trade prices 0.001 - 0.999. The same as 10th of a penny (really DAI)?

**Comment by pgebheim** [comment]
Yes.

Paul
paul@augur.net

________________________________

**Review by pgebheim — APPROVED** [review]

**Pull request merged**